src/usr/local/www/system_usermanager_addprivs.php

```diff
@@ -179,7 +179,7 @@ function get_root_priv_item_text() {
 $section->addInput(new Form_StaticText(
 	gettext('Privilege information'),
 	'<span class="help-block">'.
-	gettext('The following privileges effectively give the user root privilege ' .
+	gettext('The following privileges effectively give the user administrator-level privilege ' .
```

Comment:
s/privilege/access/ in this case to avoid awkward repetition of "privileges" in the sentence.